### PR TITLE
[1LP][RFR] Service retired

### PR DESCRIPTION
**Pull request by digitronik**
## Purpose or Intent
- Moved fixture as global 
- Automated test
- Some enhancements in ssui service pages
- Like order retired should return request object
- etc..

### PRT Run
{{pytest: cfme/tests/ssui/test_ssui_myservice.py::test_retire_owned_service --long-running}}

### CODE REVIEW

**Comment by dajoRH** [comment]
I detected some fixture changes in commit cb2c9c716f69b26224376db817d57daaabe9c68b

The *global* fixture **`user_self_service_role`** is used in the following files:
- **cfme/tests/automate/custom_button/test_service_objects.py**
- **cfme/tests/ssui/test_ssui_myservice.py**
    - *test_retire_owned_service*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by digitronik** [comment]
PRT tear down failure will fix by #9619

File: cfme/tests/ssui/test_ssui_myservice.py [hunk 1]
```diff
@@ -295,7 +298,50 @@ def test_retire_owned_service():
...
+                    service.delete()
+
+            assert service.exists
+
+            # Retire service
+            retire_request = service.retire()
+
+            @request.addfinalizer
+            def _clear_retire_request():
+                if retire_request.exists():
```
Comment (jaryn) [inline cfme/tests/ssui/test_ssui_myservice.py]: I think it this test could be made slightly more powerful by changing the `if retire_request.exists():` to `assert retire_request.exists`

**Comment by dajoRH** [comment]
I detected some fixture changes in commit e63872928be264e90072f9ff60690767dcbdfa0a

The *global* fixture **`user_self_service_role`** is used in the following files:
- **cfme/tests/automate/custom_button/test_service_objects.py**
- **cfme/tests/ssui/test_ssui_myservice.py**
    - *test_retire_owned_service*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 41c3b1b07b95069785312c06719c197411b4b8b5

The *global* fixture **`user_self_service_role`** is used in the following files:
- **cfme/tests/automate/custom_button/test_service_objects.py**
- **cfme/tests/ssui/test_ssui_myservice.py**
    - *test_retire_owned_service*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 50ea05d15da9a02c909df0b5ca45df882c994547

The *global* fixture **`user_self_service_role`** is used in the following files:
- **cfme/tests/automate/custom_button/test_service_objects.py**
- **cfme/tests/ssui/test_ssui_myservice.py**
    - *test_retire_owned_service*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by niyazRedhat** [comment]
LGTM

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
I detected some fixture changes in commit ad3c1dc59eef96c7e5c701ea8026d7b6cae1e444

The *global* fixture **`user_self_service_role`** is used in the following files:
- **cfme/tests/automate/custom_button/test_service_objects.py**
- **cfme/tests/ssui/test_ssui_myservice.py**
    - *test_retire_owned_service*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 6b7fc7d813b5b6607d22dc2251e10efb8779d806

The *global* fixture **`user_self_service_role`** is used in the following files:
- **cfme/tests/automate/custom_button/test_service_objects.py**
- **cfme/tests/ssui/test_ssui_myservice.py**
    - *test_retire_owned_service*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 04b6f3b8aafb18ce1753192140986de47ac5d1cb

The *global* fixture **`user_self_service_role`** is used in the following files:
- **cfme/tests/automate/custom_button/test_service_objects.py**
- **cfme/tests/ssui/test_ssui_myservice.py**
    - *test_retire_owned_service*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Pull request merged**